packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -49,6 +48,60 @@ const LogoContainer = styled('div')({
   },
 });
 
+function ToolpadLogo({ size = 40 }: { size?: number }) {
```

Comment:
Moved this component inside here as that's kind of how the scripts expect files to be organized, but it does take up a lot of lines...